### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by NateyLB**
### Description
Successfully fetched dummy data from our API
Integration of Google Maps API
Creation of Map Markers from dummy data
Map is centered on the US and is responsive
<p> https://trello.com/c/Nceq4oYJ </p>
<p> [x] Ran locally and passed all tests </p>

### CODE REVIEW

File: src/components/pages/Incidents/Map/Map.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import { axiosBase } from '../../../../utils/axiosBase';
+
+const Map = props => {
+  const [incidents, setIncidents] = useState([]);
+  const center = {
+    lat: 38,
+    lng: 267,
+  };
+  console.log(window.screen.width);
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Map.js]: Remove console.logs please

File: src/components/pages/Incidents/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  }
+  if (window.screen.width >= 1200) {
+    zoom = 5;
+  }
+
+  useEffect(() => {
+    axiosBase()
+      .get('/incidents')
+      .then(res => {
+        console.log(res);
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Map.js]: here too

File: src/components/pages/Incidents/Map/Map.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    return (
+      <Marker
+        key={incident.latitude + incident.longitude}
+        lat={incident.latitude}
+        lng={incident.longitude}
+        text="Incident"
+      />
+    );
+  });
+  console.log(incidents, 'STATE');
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Map.js]: Remove please

File: src/components/pages/Incidents/Map/Map.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+  useEffect(() => {
+    axiosBase()
+      .get('/incidents')
+      .then(res => {
+        console.log(res);
+        setIncidents(res.data);
+      })
+      .catch(err => {
+        console.log(err);
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Map.js:28]: This one is fine, but maybe console.error()?

File: src/components/pages/Incidents/Map/Map.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import GoogleMapReact from 'google-map-react';
+import Marker from './Marker';
+
+import { axiosBase } from '../../../../utils/axiosBase';
+
+const Map = props => {
+  const [incidents, setIncidents] = useState([]);
+  const center = {
+    lat: 38,
+    lng: 267,
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Map.js:11]: This is for centering the map?

File: src/components/pages/Incidents/Map/Map.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        setIncidents(res.data);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }, []);
+  const createMarkers = incidents.map(incident => {
+    return (
+      <Marker
+        key={incident.latitude + incident.longitude}
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Map.js:34]: Could our Markers have the same lat+long? We should be using an ID here.

File: src/components/pages/Incidents/Map/Marker.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Marker = props => {
+  return (
+    <div
+      className="marker-container"
+      style={{
+        height: '1vh',
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Marker.js]: Is there a reason for having these styles inline?

File: src/components/pages/Incidents/Map/Map.js (hunk 5 above)
Comment (NateyLB) [reply src/components/pages/Incidents/Map/Map.js:11]: Yeah, centers it on US and then the zoom gets set dynamically so the US fits within screen.

**Pull request closed without merging**